samples/topics.json

```diff
@@ -0,0 +1,183 @@
+[
+  {
+    "name": "execCmd Function",
```

Comment:
You can keep section information here, but what if we had this information in a header in the sample? I like having context in the actual sample.

```typescript
/**
--- Sample Meta
section: execCmd Function // Or you can get from the filename <section>.sample1.nut.ts
header: Testing plugin commands synchronously
usecase: I have a plugin with commands that I want to run within tests synchronously using my plugin’s `./bin/run`.
---
*/
```

Should be quick and easy to parse each sample file.

Reply:
@amphro what is the quick and easy way to get it out of the test file?  Will people be looking at the test files directly or will they see it from the samples doc?